3rdparty/7z/7zlib/7zlib.vcxproj

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Would it be possible to add crypto and 7z as seperate commits?
I'm going crazy reviewing this commit on GitHub XD

Reply:
7z is already a separate commit as for sha256, it's just 2 files :).

Reply:
Idk if this is the browser doing weird things but the project files for 7z and everything else is in this commit

Reply:
The .c/.h are in a separate commit, the 7z project was added later, as I added all the stuff to the solutions for vs it went in the same commit.